recipes/cobra/meta.yaml

```diff
@@ -7,45 +7,50 @@ package:
 source:
   fn: cobra-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/c/cobra/cobra-{{ version }}.tar.gz
-  sha256: 104e1f73f78d463849161143e8f2b98fcd7cb5ce8ee2c5b0c704ce3153036e30
+  sha256: ee74a9d585eb3fc40b3e8b6f7516bb80bb178c8413cd071c6c3189d93337cce5
 
 build:
-  number: 1
-  skip: True # [osx]
+  number: 0
+#  skip: True # [osx]
 
 requirements:
   build:
     - python
     - setuptools
```

Comment:
Looks like the actual build requirements are only `python` and `setuptools`. Anything else can be removed.